panther_analysis_tool/rule.py

```diff
@@ -94,20 +95,19 @@
 
 
 # pylint: disable=too-many-instance-attributes
-class Rule:
-    """Panther rule metadata and imported module."""
+class Detection:
```

Comment:
Minor: if this class inherits from [ABC](https://docs.python.org/3/library/abc.html#abc.ABC), it cannot be instantiated.